piet-direct2d/src/d2d.rs

```diff
@@ -167,6 +170,10 @@ fn optional<T>(val: &Option<T>) -> *const T {
     val.as_ref().map(|x| x as *const T).unwrap_or(null())
 }
 
+fn stroke_style_to_d2d(style: Option<&StrokeStyle>) -> *mut ID2D1StrokeStyle {
```

Comment:
This lives here instead of in conv because the field is private. Could move this to conv if we're ok with making the field pub.